netbox/extras/models/search.py

```diff
@@ -56,3 +59,19 @@ class Meta:
 
     def __str__(self):
         return f'{self.object_type} {self.object_id}: {self.field}={self.value}'
+
+    @property
+    def display_attrs(self):
+        """
+        Render any display attributes associated with this search result.
+        """
+        indexer = get_indexer(self.object_type)
+        attrs = {}
+        for attr in indexer.display_attrs:
```

Comment:
Does this handle if display_attrs doesn't exist (for example in an old plugin)?

Reply:
Indexers should always inherit from NetBox's base `SearchIndex` class, which defines `display_attrs`. IMO we should mandate this inheritance.